File: Journal/app/src/main/java/com/tigran/journal/MainActivity.java
package com.tigran.journal;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private String mName;
    private String mChemistryGrade;
    private String mMathGrade;
    private String mHistoryGrade;

    private TextView mNameTextView;
    private TextView mGradeTextView;

    private EditText mUserName;
    private EditText mChemistryGradeEt;
    private EditText mMathGradeEt;
    private EditText mHistoryGradeEt;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        mNameTextView = (TextView) findViewById(R.id.text_view_name);
        mGradeTextView = (TextView) findViewById(R.id.text_view_grade);

        mUserName = (EditText) findViewById(R.id.name_user);
        mChemistryGradeEt = (EditText) findViewById(R.id.chemistry_grade);
        mMathGradeEt = (EditText) findViewById(R.id.math_grade);
        mHistoryGradeEt = (EditText) findViewById(R.id.history_grade);
    }

    public void setName() {
        mName = String.valueOf(mUserName.getText());
    }

    public void setChemistryGrade() {
        mChemistryGrade = String.valueOf(mChemistryGradeEt.getText());
    }

    public void setMathGrade() {
        mMathGrade = String.valueOf(mMathGradeEt.getText());
    }

    public void setHistoryGrade() {
        mHistoryGrade = String.valueOf(mHistoryGradeEt.getText());
    }

    public void displayGrades(View v) {
        setName();
        setChemistryGrade();
        setMathGrade();
        setHistoryGrade();
        mNameTextView.setText(nameItem());
        mGradeTextView.setText(valueItem());
    }


    public String nameItem() {
        return "Name: " + "\n" +
                "Chemistry : " + "\n" +
                "Math : " + "\n" +
                "History : ";
    }

    public String valueItem() {
        return mName + "\n" +
                mChemistryGrade + "\n" +
                mMathGrade + "\n" +
                mHistoryGrade + "\n";
    }

}

File: KotlinTest/app/src/main/java/com/tigran/android/kotlintest/MainActivity.kt
package com.tigran.android.kotlintest

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.Button
import android.widget.TextView


class MainActivity : AppCompatActivity() {

    private var mTextTV: TextView? = null
    private var mChangeTextKotBtn: Button? = null
    private var mChangeTextAlexBtn: Button? = null
    private var mChangeTextColor: Button? = null
    private var mDefaultTextColor: Button? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        init()
    }

    private fun init() {
        mTextTV = findViewById(R.id.hello_from_alex)
        mChangeTextKotBtn = findViewById(R.id.change_text_kotlin_button)
        mChangeTextAlexBtn = findViewById(R.id.change_text_alex_button)
        mChangeTextColor = findViewById(R.id.change_text_color)
        mDefaultTextColor = findViewById(R.id.default_text_color)

        mChangeTextKotBtn?.setOnClickListener {
            mTextTV?.text = getString(R.string.hello_from_kotlin)
        }

        mChangeTextAlexBtn?.setOnClickListener {
            mTextTV?.text = getString(R.string.hello_from_alex)
        }

        mChangeTextColor?.setOnClickListener {
            mTextTV?.setTextColor(resources.getColor(R.color.color_text_veiw))
        }

        mDefaultTextColor?.setOnClickListener {
            mTextTV?.setTextColor(resources.getColor(R.color.default_color_text))
        }
    }

}
